README.md

```diff
@@ -99,10 +91,33 @@ type StoryPaths = {|
 |}
 
 // Retrieve and group the different stories based on their filenames
-function getFiles(options: Options): Array<StoryPaths>
+function getFiles({|
+  baseUrl?: string,              // The url for the story, defaults to:
+                                 //   'http://localhost:6006'
+  flattenFolders?: Array<string> // Folders to flatten
+                                 //   defaults to ['__snapshots__', '__screenshots__']
+  stories: any,                  // Output of require context with the stories
+  findKindAndStory?: (           // Customize "selectedKind" and "selectedStory" on the URL
```

Comment:
is there a strong need for loading stories outside of `loadStories`?

Reply:
this doesn't load the story, it allows you to customize the url for the kind/story tuple which, similarly to the other methods, is useful only when you are not using all picturebook methods